static/usage/action-sheet/theming/css-properties/demo.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Action Sheet</title>
+  <link rel="stylesheet" href="../../../common.css" />
+  <script src="../../../common.js"></script>
+  <script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@6/dist/ionic/ionic.esm.js"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@ionic/core@6/css/ionic.bundle.css" />
+
+  <style>
+    ion-action-sheet.my-custom-class {
+      --background: #F58840;
+      --backdrop-opacity: 0.6;
+      --button-background-selected: #e97223;
+      --button-color: #000000;
+      --color: #EADEDE;
+      --ion-color-danger: #fff;
+    }
+  </style>
+</head>
+
+<body>
+  <ion-app>
+    <ion-content>
+      <div class="container">
+        <ion-button onclick="presentActionSheet()">Open</ion-button>
+      </div>
+    </ion-content>
+  </ion-app>
+  <script>
+    async function presentActionSheet() {
+      const actionSheet = document.createElement('ion-action-sheet');
+      actionSheet.cssClass = 'my-custom-class';
+      actionSheet.header = 'Example header';
+      actionSheet.subHeader = 'Example subheader';
+      actionSheet.buttons = [{
+        text: 'Delete',
+        role: 'destructive',
+        data: {
+          action: 'delete'
+        }
+      },
+      {
+        text: 'Share',
+        data: {
+          action: 'share'
+        }
+      },
+      {
+        text: 'Cancel',
+        role: 'cancel',
+        data: {
+          action: 'cancel'
+        }
+      }];
+
+      document.body.appendChild(actionSheet);
```

Comment:
This should be attached to `ion-app`. Alternatively, you can use the alertController which will do this for you. (same goes for the other instances of `document.body.appendChild`)

Reply:
Hmm looks like alert also does `document.body`: https://ionicframework.com/docs/api/alert

I'd be fine doing `document.body` for now to be consistent and then revisiting the best practice later.

Reply:
Honestly - I referenced `ion-alert` examples when creating these examples. I can update both to match a specific usage pattern, if we have one.